.github/workflows/functional-test-cloud.yaml

```diff
@@ -758,6 +758,12 @@ jobs:
             }
           }
           EOF
+      - name: Restore Bicep artifacts before running functional tests
+        run: |
+          # Restore Radius Bicep types
+          bicep restore ./test/functional-portable/corerp/cloud/resources/testdata/corerp-azure-connection-database-service.bicep --force
+          # Restore AWS Bicep types 
+          bicep restore ./test/functional-portable/corerp/cloud/resources/testdata/aws-s3-bucket.bicep --force
```

Comment:
Haven't look into this much in depth, can you help me understand why we need to provide path to specific test files? And why just these two?

Reply:
This is the command being used: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-cli#restore

The error we're getting is that the bicep artifacts can't be restored before the bicep build command starts running. This command forces a restore before any builds start so the artifacts can be downloaded successfully. The exact files chosen to run the command on aren't necessarily important but we need 1 that uses the Radius extension and 1 that uses the AWS extension so we can restore both bicep artifacts.

Reply:
The build command automatically calls the restore command, but since we're seeing flaky failures with this, we'll restore first and then rely on the build command

Reply:
Makes sense, thanks for sharing the details. Could you add your note about the kind of files needed in the comments? It will help in directing us which files can be used here if we ever were to delete these two files.

`The exact files chosen to run the command on aren't necessarily important but we need 1 that uses the Radius extension and 1 that uses the AWS extension so we can restore both bicep artifacts.`